Automattic/jetpack
# Modules: Make the generated module-headings.php pass PHPCS

#### Changes proposed in this Pull Request:

The file was edited to pass phpcs in #17762, so now every `yarn build`
overwrites it with a changed version. Attempting to commit that changed
version will wind up whining about all the phpcs failures in the
generated version, and fixing them (mostly with phpcbf, some manually)
will wind up as a no-op.

This patch updates tools/build-module-headings-translations.php to fix
some of the errors and emit `phpcs:disable` for two others that isn't
worth trying to generate the way the WordPress coding standards want.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Run `yarn build`. `git status` should not indicate any change to module-headings.php.
* Run `composer php:lint modules/module-headings.php`. It should report no errors or warnings.

#### Proposed changelog entry for your changes:


* None needed.
